airflow/contrib/operators/dataflow_operator.py

```diff
@@ -118,6 +118,8 @@ def __init__(
             delegate_to=None,
             poll_sleep=10,
             job_class=None,
+            check_if_running=None,
+            multiple_jobs=None,
```

Comment:
Can we have some documentation on the `multiple_jobs ` functionality?

Reply:
You can have a pipline that spans on more than one job, this will wait until all jobs have finished

Reply:
Documentation in the docstring in code, in in a comment in github :)